.github/ISSUE_TEMPLATE/release.yml

```diff
@@ -28,74 +28,89 @@ body:
+        - [ ] Remove the additional fields (`Remotes`) from the DESCRIPTION file where applicable.
         - [ ] Make sure that the minimum dependency versions are updated in the DESCRIPTION file for the package.
-          - [ ] Increase versioned dependency on {package name} to >=X.X.X.
-        - [ ] Create a pull request to make necessary bug fixes/changes (add "[skip vbump]" in the PR title), and after merging the PR, tag the update(s) as a release candidate v < intended release version > -rc < release candidate iteration > on the main branch. Note that tags are created in GitHub and synchronized with GitLab automatically.
-        - [ ] The package is submitted for internal validation by Release Coordinator (Applicable only for regulatory release).
-        - [ ] Address any feedback (internal validation/user testing), retag the package as a release candidate vX.X.X-rc(n+1). Repeat the submission for internal validation if necessary.
-        - [ ] Get the package validated (Applicable only for regulatory release).
-        - [ ] If the additional fields were removed, add them back in a separate PR, and then merge the PR back to main (add "[skip vbump]" in the PR title).
-        - [ ] Create a git tag with the final version set to X.X.X on the main branch.
-        - [ ] Update downstream package dependencies to (>=X.X.X) in {package name}.
-  - type: textarea
-    id: testing
-    attributes:
-      label: Testing
-      description: Summary of testing activities - integration tests, UAT, other.
-      placeholder: Tests results
-      value: |
-        - [ ] Integration tests results - accepted (Internal Compute Environments/Validation Pipelines).
-        - [ ] UAT results - accepted.
-        - [ ] Shiny apps test results - accepted (Applicable only for Shiny apps).
-        - [ ] Necessary testing on target environment - performed (up to ETL).
-  - type: textarea
-    id: feedback
-    attributes:
-      label: Release Feedback
-      description: Feedback received from internal validation/UAT testers.
-      placeholder: Feedback to be implemented after submission for internal validation/testing.
-      value: |
-        - [ ] Fix 1
-        - [ ] Enhancement 1
-        - [ ] Defect 1
+          - [ ] Increase versioned dependency on {package name} to >=X.Y.Z.
+        - [ ] Commit your changes and create the PR on GitHub (add "[skip vbump]" in the PR title). Add all updates, commit, and push changes:
+        `# Make the necessary modifications to your files
+        # Stage the changes
+        git add <files your modified>`
+        # Commit the changes
+        git commit -m "[skip vbump] <Your commit message>"
+        git push origin release-candidate-vX.Y.Z`
+
+        ### Test the release
+        
+        - [ ] Execute the manual tests on Shiny apps that are deployed on various hosting providers (Posit connect and shinyapps.io) - track the results in GitHub issue (Applicable only for frameworks that use Shiny).
```

Comment:
do we have a designated posit cloud account for this? or should the person whoever is doing this to use their own account? for both posit cloud and shinyapps.io

Reply:
i think definitely should also test on

* connect-pam.workbench.roche.com
* https://connect-stage.workbench.roche.com/connect/
* https://connect.workbench.roche.com/connect/